extjsdk/build.gradle

```diff
@@ -93,6 +93,7 @@ dependencies {
 task fatJar(type: Jar) {
     appendix = 'fat'
     from sourceSets.main.output
+    duplicatesStrategy 'include'
```

Comment:
Gradle 7.x gives an error by default on duplicate files in any copy-derivative task.  Have to specify how to handle duplicates.  We choose to allow them to override one another...